content/automate/ManageIQ/Transformation/StateMachines/TransformationPlanRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -1,5 +1,9 @@
-#
-# Description: Placeholder for service request validation
-#
-prov = $evm.root['miq_request']
-prov.source_vms.each { |vm| prov.approve_vm(vm) if prov.validate_vm(vm) }
+request = $evm.root['miq_request']
+
+unless request.validate_conversion_hosts
+  $evm.object['reason'] = 'No conversion host configured'
+  request.message = 'No conversion host configured'
+  exit MIQ_ABORT
```

Comment:
@tinaafitz @mkanoor Should this remain `exit` or is it better to use `raise`?

Reply:
@gmcculloug It doesn't hurt to have the exit there, we solved the spec problem with the approach listed here
https://github.com/ManageIQ/manageiq-content/blob/166a3df922e31953f63db76dd644df734fd1a40d/spec/content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object_spec.rb#L81

Which allows us to test different exit codes. So if there are specs for this change it should follow a similar pattern if the method is being tested directly outside of the DRb and Automate Engine framework.